docs/build/advanced-concepts/flix.md

```diff
@@ -8,18 +8,49 @@ Flow Interaction Templates is a standard for how contract developers, wallets, u
 
 Interaction Templates provide a way to use and reuse existing scripts and transactions, as well as to provide more metadata such as a human-readable title and description of what the transaction or script will do, which can be used by the developer as well as the user of the application. 
 
-By using FLIX transactions and scripts, develoeprs don’t have to write their own for common operations!
+By using FLIX transactions and scripts, developers don’t have to write their own for common operations!
 
 Read more about the design and purpose of FLIX in the [FLIP](https://github.com/onflow/flips/blob/main/application/20220503-interaction-templates.md)
 
 ## Using FLIX
 
-Flow makes FLIX available through an API available at flix.flow.com. Other community run APIs are available, such as flix.ecdao.org. 
+Flow makes FLIX available through an API available at flix.flow.com. 
```

Comment:
should `flix.flow.com` be a link?

Reply:
nope, since it's a api and needs full url, which is lower in the markdown